.github/workflows/publish-pypi.yml

```diff
@@ -49,28 +49,22 @@ jobs:
         with:
           submodules: true
 
-      - name: "Setup Python environment"
-        run: |
-          python -m pip install --user pipx
-          python -m pipx install -f poetry==${{ needs.prepare-environment.outputs.POETRY_VERSION }}
-          poetry config virtualenvs.create true --local
-          poetry env use 3.12
-        env:
-          PIPX_DEFAULT_PYTHON: ${{ steps.python.outputs.python-path }}
-
-      - name: "Install Dependencies"
-        run: "poetry install"
-        working-directory: ${{ matrix.package_dir }}
+      - name: Install uv
+        uses: astral-sh/setup-uv@v7
+        with:
+          version: ${{ needs.prepare-environment.outputs.UV_VERSION }}
+      - name: Install Dependencies
+        run: uv sync --all-groups --all-extras
       - name: "Add PyPI secret"
-        run: "poetry config pypi-token.pypi ${{ secrets.PYPI_TOKEN }}"
+        run: "uv config token pypi ${{ secrets.PYPI_TOKEN }}"
```

Comment:
It looks like this `uv config` came from some AI hallucination in the SDK repo..  

```bash
root@c59566bd89ad:/# uv self version
uv 0.9.10
root@c59566bd89ad:/# uv config
error: unrecognized subcommand 'config'

Usage: uv [OPTIONS] <COMMAND>

For more information, try '--help'.
root@c59566bd89ad:/#
```

https://github.com/opsmill/infrahub-sdk-python/pull/637#discussion_r2535069801